tests/requirements/oracle.txt

```diff
@@ -1 +1 @@
-oracledb >= 1.3.2
+oracledb >= 2.3.0
```

Comment:
@suraj-ora-2020 there's more work required to bump the minimum version (see 9946f0b0d9356b55e819f861b31615fa5b548f99 as an example)

We would need to: 
- update the docs saying ```Version xxx or higher of the `oracledb`_ Python driver is required``` in `docs/ref/databases.txt`
- add a release note along the lines of ```* The minimum supported version of ``oracledb`` is increased from 1.3.2 to
  xxxx.```

Reply:
Done.